guides/common/modules/proc_synchronizing-a-content-view-to-a-smartproxy.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Synchronizing_a_Content_View_to_a_{smart-proxy-context}_{context}"]
+= Synchronizing a content view to a {SmartProxy}
+
+In the {ProjectWebUI}, you can only synchronize all selected lifecycle environments simultaneously.
+If you need to synchronize smaller items, such as individual lifecycle environments, single content views, and single repositories, use the Hammer CLI.
+
+.CLI procedure
+* Synchronize the content view to your {SmartProxy} by using Hammer:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ {hammer-smart-proxy} content synchronize --content-view "my content view name"
+----
+
+.Additional resources
+* For more information about the command, run `{hammer-smart-proxy} content synchronize --help`.
```

Comment:
Personally, I would put this info in the introduction, after line 5.

Reply:
Can you explain why? Because we have been including --help references in additional resources and so does the RHEL team (for man pages).

Reply:
It's purely a personal preference. Here are my reasons:

- "Additional resources" can be used too often. Example: assembly with "Additional resources", with 5-10 xrefs, after **every single module**.
- "Additional resources" can turn into a junk drawer for useless info. Example: "See the next procedure", with a link to the next module.
- Users tend not to read the entire file, so they are likely to miss anything at the bottom of the page.
- Sections with only one item look 'noisy' to me. "Prerequisites" and "Procedure" are OK because they are standard.

Reply:
I'm not saying this should be changed. I'm just saying that I personally don't like it. However, if Foreman docs and RHEL are doing this consistently, then it should be kept.

Reply:
> I'm not saying this should be changed. I'm just saying that I personally don't like it. However, if Foreman docs and RHEL are doing this consistently, then it should be kept.

Ok, I'd say we should stick to Additional resources, then. That would be in compliance with RH SSG too (the guide explicitly talks about [including man page references in additional resources](https://redhat-documentation.github.io/supplementary-style-guide/#man-pages)).

>     Users tend not to read the entire file, so they are likely to miss anything at the bottom of the page.

I'd argue that this is arbitrary and varies from user to user. Some users might actually "wake up" towards the end and start focusing again. But please, if you know about any data from CCS doc usability sessions that you could share to support this, that would be great.

But actually, even if this is the case, it's quite alright. The main part of the procedure should cover whatever users actually need to do. The idea behind including man pages/--help pages in additional resources is that it's an optional resource that users can access in case they want to see "additional" explanations or options. Hence, "additional" resources